protocol/contracts/lease/src/position/spec/mod.rs

```diff
@@ -1163,24 +1162,28 @@ mod test_validate_close {
 
     #[test]
     fn amount_as_min_transaction() {
-        let spec = position(100, 85, 15);
-        let result_1 = spec.validate_close_amount(15.into(), price(1, 1));
+        let spec = spec(85, 15);
+        let asset: Coin<TestCurrency> = 100.into();
```

Comment:
[everywhere in this mod] It's too much to specify each time the type. Alternatives?